engine/src/bibtex.rs

```diff
@@ -926,9 +869,9 @@ unsafe extern "C" fn bib_id_print() {
         puts_log(b"You\'re missing \x00" as *const u8 as *const i8);
     } else if scan_result as i32 == 2i32 {
         /*other_char_adjacent */
-        printf_log(
-            b"\"%c\" immediately follows \x00" as *const u8 as *const i8,
```

Comment:
The original format specifier uses %c. The current behavior changes since `buffer` is not a `String`.

Reply:
Changed all instances like this to print char